app/views/public/privacy_policy/index.html.haml

```diff
@@ -0,0 +1,225 @@
+
+      %p.govuk-body
+        How we protect your data and keep it secure
+
+      %p.govuk-body
+        We are committed to doing all that we can to keep your data secure. We have set up systems and processes to prevent unauthorised access or disclosure of your data - for example, we protect your data using varying levels of encryption.
+
+      %p.govuk-body
+        We also make sure that any third parties that we deal with keep all personal data they process on our behalf secure.
+
+      %h2.govuk-heading-m
+        Links to and from other websites
+
+      %p.govuk-body
+        RODA  contains links to other websites.
+
+      %p.govuk-body
+        This privacy notice only applies to RODA, and does not cover other websites that we link to.
+
+      %h3.govuk-heading-s
+        Following a link to another website
+
+      %p.govuk-body
+        If you go to another website from this one, read the privacy policy on that website to find out what it does with your information.
+
+      %h3.govuk-heading-s
+        Following a link to RODA from another website
+
+      %p.govuk-body
+        If you come to RODA from another website, we may receive personal information from the other website. You should read the privacy policy of the website you came from to find out more about this.
+
+      %h2.govuk-heading-m
+        Your rights
+
+      %p.govuk-body
+        You have the right to request:
+
+      %ul.govuk-list.govuk-list--bullet
+        %li information about how your personal data is processed
+        %li a copy of that personal data
+        %li that anything inaccurate in your personal data is corrected immediately
+
+      %p.govuk-body
+        You can also:
+
+      %ul.govuk-list.govuk-list--bullet
+        %li raise an objection about how your personal data is processed
+        %li request that your personal data is erased if there is no longer a justification for it
+        %li ask that the processing of your personal data is restricted in certain circumstances
+
+      %p.govuk-body
+        If you have any of these requests, get in contact with our RODA Support Team.
+
+      %h2.govuk-heading-m
+        Contact us or make a complaint
+
+      %p.govuk-body
+        Contact the RODA Support Team if you:
+
+      %ul.govuk-list.govuk-list--bullet
+        %li have a question about anything in this privacy notice
+        %li think that your personal data has been misused or mishandled
+
+      %address.govuk-body
+        RODA Support Team
+        %br/
+        1 Victoria Street
+        %br/
+        London
+        %br/
+        SW1H 0ET
+        %br
+        United Kingdom
+
+        %p.govuk-body
+          Email:
+          = link_to "enquiries@beis.gov.uk", "mailto:enquiries@beis.gov.uk", class: "govuk-link"
+
+        %p.govuk-body
+          Telephone:
+          = link_to "20 7215 5000", "tel:+442072155000", class: "govuk-link"
```

Comment:
`20 7215 5000` is not recognised when I ring it. I think we need to include an additional zero here: 020 7215 5000

Reply:
Great spot! Thanks!